ush/ioda/bufr2ioda/bufr2ioda_adpsfc_prepbufr.py

```diff
@@ -80,13 +80,16 @@ def bufr_to_ioda(config, logger):
     q.add('latitude', '*/YOB')
     q.add('longitude', '*/XOB')
     q.add('obsTimeMinusCycleTime', '*/DHR')
-    q.add('heightOfStation', '*/Z___INFO/Z__EVENT{1}/ZOB')
+    q.add('height', '*/Z___INFO/Z__EVENT{1}/ZOB')
     q.add('pressure', '*/P___INFO/P__EVENT{1}/POB')
 
-#   # Quality Infomation (Quality Indicator)
+    # Quality Marker
     q.add('qualityMarkerStationPressure', '*/P___INFO/P__EVENT{1}/PQM')
     q.add('qualityMarkerStationElevation', '*/Z___INFO/Z__EVENT{1}/ZQM')
 
+    # ObsError
+    q.add('obsErrorStationPressure', '*/P___INFO/P__BACKG{1}/POE')
```

Comment:
No, most of the time it gets overwritten. I need it for my ADPSFC/SFCSHP where it will eventually get overwritten but it was needed to run some of the end-to-end tests so I was told to put it in and if it gets overwritten then so  be it.